src/Microsoft.Extensions.Testing.Abstractions/project.json

```diff
@@ -12,7 +12,8 @@
       "target": "project"
     },
     "Microsoft.DiaSymReader": "1.0.8",
-    "Microsoft.DiaSymReader.Native": "1.4.0-rc2"
+    "Microsoft.DiaSymReader.Native": "1.4.0-rc2",
+    "System.Reflection.Metadata": "1.4.1-beta-24410-02"
```

Comment:
Does TestingAbstractions use anything in System.Runtime?  System.Collections?  I don't see those dependencies listed...

I truly believe that all we are doing here is adding more work for the future, and opening a door for these versions to get out of sync.  The projects live next to each other inside the same repo, and their dependencies should be in lock step. Defining it in a single place ensures that.